build.gradle

```diff
@@ -423,16 +424,17 @@ task stopEC2(type: AWSLambdaInvokeTask) {
 //////////////////////////////////////////////////////////////////////
 task generateConfig {
     doLast {
-        if (!project.file('build/tscfg-0.8.1.jar').exists()) {
+        def tscfgJarFile = project.file('build/tscfg-'+tscfgVersion+'.jar')
+        if (!tscfgJarFile.exists() || !tscfgJarFile.isFile()) {
```

Comment:
checkin if it exists is redundant because `isFile` already check if it exists and is a file. the following should be enough:
```Groovy
if (!tscfgJarFile.isFile()) {
```